### ISSUE TITLE
Manual for installation alongside Wireguard on Debian

### ISSUE THREAD

**Issue by HellStorm666** [ngoduykhanh/wireguard-ui#363]
Hi all,

Would love to use this wireguard-ui, looks awesome.
However, I tried to install it alongside my Wireguard install on Debian, but it isn't working.
I do get the web interface, it does show some stuff, but no clients for instance on the already configured Wireguard (wg0.conf).
Also, when I add a new client, it does show up, but the new clients don't work.

Can someone make a good manual on how to install wireguard-ui if you have Wireguard already installed?
Of course, with running as a service and reloading Wireguard if config is changed etc.

**Comment by ZMJGMADHPXWT** [comment]
Hi. 
Same here, I'd like to get an instruction how to use wireguard-ui with existing WireGuard server. But the difference is in the OS - I have CentOS. I have a coule of existing users and my server works fine, but I'd like to be able to get QR and archives for old and new VPN clients.


Thanks in advance.

**Comment by HellStorm666** [comment]
Well, in the mean time i've found a way to get it to work.
I made a backup of the wg0.conf, public key and private key.
Then I ran this script: https://gitlab.com/snax44/wireguard-ui-setup
Made the clients I had new and clicked apply config.
Then I eddited the .json files for the clients (/opt/wgui/db/clients) with the values I had in wg0.conf, changed the private key and public key in the corrrect json files. Pressed apply config again, and it worked.
